taxi/views.py

```diff
@@ -36,6 +39,17 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_queryset(self):
+        query = self.request.GET.get("q")
```

Comment:
It is a good practice to explicitly indicate what parameter is used for filtration among query parameters. E.g., there could be search by multiple query parameters simultaneously. To prevent naming confusion using `q`, use `name` as a query parameter here and corresponding parameters in the other views. You could create separate search forms for each case to achieve that. Adjust code in all places to achieve that